src/helpers/utils.ts

```diff
@@ -149,3 +151,27 @@ export const paramsReducer = (
       return removeParams(existingParams, newParams);
   }
 };
+
+/**
+ * Stringifies value to '' if the value is null
+ */
+export const replaceNullWithEmptyString = (_: any, value: any) =>
+  value === null ? '' : value;
```

Comment:
question (blocking): I don't think this function needs two arguments, look like only `value` is being used.

```suggestion
suggestion (blocking): export const replaceNullWithEmptyString = (value: any) =>
  value === null ? '' : value;
```

Reply:
comment:

The `JSON.stringify()` function [accepts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#the_replacer_parameter) a `replacer` array or function. 

```js
JSON.stringify(value, replacer)
```

The function has two arguments, `key` and `value`. We pass `replaceNullWithEmptyString` as the replacer function to JSON.stringify. 

We only need `value`, so we use `_` to signify we don't intend to need or use the first argument, `key`.

This is really just to keep `stringifyJsonFields` readable, but it can come in handy wherever else you use `JSON.stringify`.

Reply:
TIL thanks